src/executables/uber-time.js

```diff
@@ -8,13 +8,19 @@ import CommandExecutionService from '../services/CommandExecutionService';
 
 const service = new CommandExecutionService();
 
+const address = program.args[0].trim().toString();
 program
   .arguments('<address>')
   .action((address) => {
+    if (!address.length) {
```

Comment:
since the pattern has been to delegate validation to the `CommandExecutionService` do you think it makes sense to move this validation to that service?